hilti/toolchain/src/compiler/driver.cc

```diff
@@ -642,10 +642,8 @@ Result<Nothing> Driver::run() {
         return Nothing();
 
     } catch ( const std::exception& e ) {
-        std::cerr << util::fmt("[fatal error] terminating with uncaught exception of type %s: %s",
-                               util::demangle(typeid(e).name()), e.what())
-                  << std::endl;
-        exit(1);
+        return result::Error(
+            fmt("terminating with uncaught exception of type %s: %s", util::demangle(typeid(e).name()), e.what()));
```

Comment:
This now doesn't know anymore whether it'll actually abort.
```suggestion
            fmt("uncaught exception of type %s: %s", util::demangle(typeid(e).name()), e.what()));
```